lib/sequent/test/command_handler_helpers.rb

```diff
@@ -84,17 +89,26 @@ def deserialize_events(events)
 
       end
 
-      def given_streams_with_events *streams_with_events
-        @event_store.given_events(streams_with_events)
+      def given_events(aggregate_class, aggregate_id, *events)
+        @event_store.given_events [Sequent::Core::EventStream.new(aggregate_type: aggregate_class.to_s, aggregate_id: aggregate_id, snapshot_threshold: 30, stream_record_id: 1), events]
       end
 
+
       def when_command command
         raise "@command_handler is mandatory when using the #{self.class}" unless @command_handler
         raise "Command handler #{@command_handler} cannot handle command #{command}, please configure the command type (forgot an include in the command class?)" unless @command_handler.handles_message?(command)
-        @command_handler.handle_message(command)
+        begin
+          @command_handler.handle_message(command)
+        rescue Sequent::Core::DomainError => e
```

Comment:
I'm scoping to Sequence::Core::DomainError in order to make other kind of errors to flows throw the test and interrupt the test execution.
But yes I can make it without scoping, I'll change it.